cypress-ci.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Why do we need a different cypress for the CI ? Why not just run next on port 3001 in the CI too ?

Reply:
Deleted in a7c55e9663e2c8038cecd288c13623b66123b621